.github/workflows/e2e-verify-basicinfo.yml

```
@@ -0,0 +1,166 @@
+        uses: shivammathur/setup-php@7c071dfe9dc99bdf297fa79cb49ea005b9fcadbc  # v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: redis
+
+      - name: Setup Node.js
+        uses: actions/setup-node@48b55a011bda9f5d6aeb4c2d9c7362e8dae4041e  # v6.4.0
+        with:
+          node-version: 20
+
+      - name: Initialize Composer
+        uses: ./.github/actions/composer
```

actionlint (1.7.12)

[error] description is required in metadata of "" action at "/home/jailuser/git/.github/actions/composer/action.yml"

(action)